pychunkedgraph/app/meshing/v1/routes.py

```diff
@@ -89,3 +91,10 @@ def handle_get_manifest(table_id, node_id):
 @remap_public(edit=True)
 def handle_remesh(table_id):
     return common.handle_remesh(table_id)
+
+
+@bp.route("/table/<table_id>/clear_manifest_cache/<node_id>", methods=["POST"])
+@auth_requires_permission("admin")
+def handle_merge_admin(table_id, node_id):
```

Comment:
bad function name. Should be something like `handle_clear_manifest_cache`

Reply:
ah, copy pasted the function and decorators..